sync/sync_head_test.go

```diff
@@ -138,3 +168,25 @@ func (t *wrappedGetter) GetRangeByHeight(
 	// TODO implement me
 	panic("implement me")
 }
+
+type errorGetter struct{}
+
+func (e errorGetter) Head(ctx context.Context, h ...header.HeadOption[*headertest.DummyHeader]) (*headertest.DummyHeader, error) {
+	time.Sleep(time.Millisecond * 1)
```

Comment:
Do we really need this? (not against, just asking).

Reply:
Otherwise the test doesnt fail